src/diffusers/schedulers/scheduling_edm_dpmsolver_multistep.py

```diff
@@ -107,6 +110,7 @@ def __init__(
         lower_order_final: bool = True,
         euler_at_final: bool = False,
         final_sigmas_type: Optional[str] = "zero",  # "zero", "sigma_min"
+        noise_preconditioning_strategy: str = "log",
```

Comment:
umm I need to think about this more, but I think we can use a `use_consine_scheduler` to configure the difference  
![Screenshot 2024-04-25 at 22 12 08](https://github.com/user-attachments/assets/a468092a-180c-4f3c-9a34-de8682636fb9)

Reply:
> it seems that it's not easy to make it work with the non-EDM version?
None that I've seen!